.vscode/launch.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
I am not sure if we should add this here.. perhaps sample configs in another directory or `HACKING.md` along with other notes/pointers?

Reply:
Ah, this was a mistake in all honesty. But yes i think having sample configs and a HACKING.md would be good. I'll remove this from the PR and have a separate PR.